src/components/media-player/volume-presets-modal.js

```diff
@@ -52,8 +52,7 @@ function VolumePresetsModal({ volumePresets, isOpen, onOk, onClose }) {
       ...oldPresets,
       {
         name: '',
-        mainTrack: 1,
-        secondaryTracks: new Array(volumePresets[0].secondaryTracks.length).fill(1)
+        tracks: new Array(volumePresets[0].tracks.length).fill(1)
```

Comment:
Tiny thing, too: I know this code was there before, but maybe we can switch to our more often used (and more modern) pattern of `Array.from({ length: volumePresets[0].tracks.length }, x => 1)` because the `Array` constructor has two easy to confuse overloads (one where you set the length, one where you set the initial elements) which makes it very ambiguous.

Reply:
Nicely spotted, changed. Thanks!